LoopKitUI/Extensions/DeviceStatusHighlight+Color.swift

```diff
@@ -10,7 +10,11 @@ import LoopKit
 
 extension DeviceStatusHighlight {
     public var image: UIImage {
-        return UIImage(systemName: imageSystemName)!
+        if let image = UIImage(frameworkImage: imageName) {
+            return image
+        } else {
+            return UIImage(systemName: imageName)!
```

Comment:
[nit] it bugs me a bit that this force-unwraps here.  Because these names come from plugins, It means that a plugin might crash the app.  Would you consider either making this return an `Optional` (so that the calling code can at least fall back to something reasonable) or changing it to a `func` that throws an error (your choice)?  Not blocking, I'd be fine with changing it in a later PR.

Reply:
Sure. I'll update to return an optional.